services/metadactyl-clj/src/metadactyl/user.clj

```diff
@@ -17,9 +15,9 @@
   (log/debug user-attributes)
   (let [uid (user-attributes :user)]
     (if (empty? uid)
-      (throw+ {:error_code common-errors/ERR_NOT_AUTHORIZED,
-               :user (dissoc user-attributes :password),
-               :message "Invalid user credentials provided."}))
+      (throw+ {:type :clojure-commons.exception/bad-request-field
+               :error "Invalid user credentials provided."
+               :user (dissoc user-attributes :password)}))
```

Comment:
Yeah, exactly. Choose the keys you do want rather than removing the keys you don't. That way if for some reason we add `:seekritTokenNeverToBeExposed` this code doesn't need to be updated (that is, we don't have to remember that this code needs to be updated, which is the real issue). I don't know exactly what's relevant to this error -- possibly just `shortUsername`, since it's about credentials and obviously we're not including the password. But including the full plus short, name, and email seems like it can't hurt.